src/theme/types.ts

```diff
@@ -10,20 +12,32 @@ export type {
     SemanticColourSetOptions,
 } from "./colour-semantic/types";
 
+export type { FontSet, FontSetOptions } from "./font/types";
+
+export type {
+    TypographySet,
+    TypographySetOptions as TypoSetOptions,
```

Comment:
remove the alias, let's standardise to use `Typography` instead of `Typo`
```suggestion
    TypographySetOptions,
```